app/views/terms/show.html.erb

```diff
@@ -1,10 +1,10 @@
 <header class="page-header">
   <h1>
-    <%= @document.title %>
+    <%= t(@document.title) %>
```

Comment:
I'm still thinking on that... Relying on fixed translation keys ala `t('terms.title')` seems simpler, but then we'll "lose" this automatic check by `updated_at`:

https://github.com/coopdevs/timeoverflow/blob/1820729332b7faff535c74dc26e456cba8deae14/app/controllers/application_controller.rb#L65

The current solution will require to add new keys for any terms udpate?

Another options (we should discuss them with @sseerrggii too): 
- `json_translate` gem, as we do for categories in `translates :name`
- Go with static keys and have a CONSTANT (or similar) with the timestamps of terms udpates

Reply:
@franpb14 as we discussed via chat, probably moving to DB translations, via the `json_translate` gem, seems to fit better for this use-case.